readme-template.md

```diff
@@ -0,0 +1,107 @@
+# Repo Title
+
+<!-- Not all of these are essential but keep this in mind for the repo you are using -->
+![Build badge]() ![Test Coverage]() ![npm package]()
+
+<!--Short sentence on what the repo does-->
+
+<!-- possibly list experts in this repo for who to go to when encountering a problem-->
+
+##Contents
+* [Further Documentation](#documentation)
```

Comment:
I dunno, I am in 2 minds with this. I think for the initial run through of the repo yes, but after you have it set up, your're most likely going to be reading other documentation. We can by all means change it later or if more people disagree on the placement we can shift it :)